frontend/src/api/requests.ts

```diff
@@ -7,7 +7,7 @@
  * Custom type definition for the HTTP methods handled by this module.
  */
 type Method = "GET" | "POST" | "PUT" | "PATCH" | "DELETE";
-const API_BASE_URL = process.env.NEXT_PUBLIC_API_BASE_URL ?? "/api";
+const API_BASE_URL = process.env.NEXT_PUBLIC_API_BASE_URL ?? "http://localhost:4000/api";
```

Comment:
This needs to be what it was previously `const API_BASE_URL = process.env.NEXT_PUBLIC_API_BASE_URL ?? "/api";`. In your frontend env file add the following variable `NEXT_PUBLIC_API_BASE_URL='http://localhost:4000/api'`

Reply:
Ok, I just changed it back